vioscsi/helper.c

```diff
@@ -111,7 +112,9 @@ ENTER_FN_SRB();
         }
```

Comment:
Just a curiousity, if we enqueue work on Line98, wouldn't it be "faster" to do kick processing immediately afterwards, so that we kick the underlying datapath as soon as physically possible?

Or is the idea that we want to release VioScsiVQUnlock such that another IO could possibly be submitted?